pontoon/base/migrations/0009_change_pontoon_users_emails.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.13 on 2020-08-28 08:49
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def update_base_user_email(apps, schema_editor):
```

Comment:
Maybe `update_system_user_email`?

Reply:
Okay , also if i change that i  would also need to update the code as
```
operations = [
        migrations.RunPython(
            code=update_system_user_email, reverse_code=revert_system_user_email,
        ),
    ]

```

Reply:
Exactly. :-)

Reply:
May i ask the reason for changing the function name to
```update_system_user_email
```
instead of 
```update_base_user_email
```
As we are making changes to the base app i cannot understand this change.

Reply:
The current name implies we're changing the email address of the "base user". But we're actually changing built-it (preinstalled) Pontoon users AKA "system users". So the new method name more accurately describes what we're doing.

Reply:
Ohh, Got it(we are changing email domain of system accounts) thanks .